examples/README.md

```diff
@@ -0,0 +1,20 @@
+Run "bash run_gate_benchmark.sh" in the terminal. The script will automatically build the gate_benchmark project.
+It will set the CXX and CMAKE_CXX_FLAGS environment variables to "which clang++" and "-O3".
```

Comment:
It would be great if this were user-selectable, rather than set in the script (even as a command-line argument). Other potential options could be `g++`, `icpc`, `icpx`, `nvc++` and others.

Reply:
The compiler name now has to be specified as a command-line argument after `run_gate_benchmark.sh` script.